### DISCUSSION TITLE
These CDP instructions may have hidden dangers.

### DISCUSSION THREAD

**Discussion by chuzilaolin** [apify/fingerprint-suite#331] [Q&A]
![image](https://github.com/user-attachments/assets/cecea87d-6e4f-40b1-b7ea-ca23821586ba)

**This CDP command may be deprecated in the future, which would render this fingerprinting approach obsolete.**

![image](https://github.com/user-attachments/assets/0817a7e9-3b3f-45e0-b3f3-4ddd683e2006)

**Comment by barjin** [comment]
Hello, and thank you for your interest in this project.

Interesting, neither the `tip-of-tree`, nor the `stable` version of the CDP documentation we use lists this method as `deprecated` / `experimental` ([link here](https://chromedevtools.github.io/devtools-protocol/tot/Emulation/#method-setDeviceMetricsOverride))  Would you mind sharing a link to the documentation you took the screenshot from? Thanks!

**Comment by barjin** [comment]
Oh, right, I see it now. This method is deprecated for `Page`, the one I'm linking is from the `Emulation` domain. I'll investigate and see what can be done about this. Thank you again for sharing this with us!

**Reply by chuzilaolin** [reply]
Thank you very much for your response. I will continue to keep an eye on this project going forward.
